### PR TITLE
Observability upgrade for installer scripts

### PR DESCRIPTION
**Pull request by shaynafinocchiaro**
# Description
This PR updates CSM for Observability upgrade documentation to support installer scripts. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/151 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/_index.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ The CSI Drivers by Dell EMC implement an interface between [CSI](https://kuberne
 | Ubuntu        |       20.04      |       20.04         |       18.04, 20.04      |        18.04, 20.04      |          20.04     |
 | CentOS        |     7.8, 7.9     |      7.8, 7.9       |     7.8, 7.9     |      7.8, 7.9     |     7.8, 7.9     |
 | SLES          |        15SP3        |        15SP3        |       15SP3      |         15SP3     |       15SP3      |
-| Fedora Core OS|        no        |         5.x       |        no        |         no        |        no        |
```
Comment (tdawe) [inline content/docs/csidriver/_index.md:24]: Verify your branch is rebased with latest from the release-2.1 branch. There is a commit and changes in this PR that are not related to the observability changes.

Comment (shaynafinocchiaro) [reply content/docs/csidriver/_index.md]: Rebased with release-1.2 branch

**Review by alikdell — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]
